### ISSUE TITLE
question: twitter cards

### ISSUE THREAD

**Issue by fpigeonjr** [maxpou/gatsby-starter-morning-dew#38]
So I am excited to implement the social media cards but am not getting them to show.

![Imgur](https://i.imgur.com/0V8C0VJ.png)

## steps to recreate
per https://maxpou.github.io/gatsby-starter-morning-dew/gatsby-starter-morning-dew-v1-1
1. `yarn develop`
1. `yarn generatePostPreviewImages`

   Everything seemed to work ok in the terminal

   ```bash

    myBlog git/master*
    ❯ yarn generatePostPreviewImages
    yarn run v1.10.1
    $ node ./scripts/generatePostPreviewImages.js
    Created /Users/fpigeonjr/Code/myBlog/content/posts/2017-02-24-hello-world/hello-world-fb.png
    Created /Users/fpigeonjr/Code/myBlog/content/posts/2017-02-24-hello-world/hello-world-tw.png
    Created /Users/fpigeonjr/Code/myBlog/content/posts/2017-02-25-Goals/goals-for-2017-fb.png
    Created /Users/fpigeonjr/Code/myBlog/content/posts/2017-02-25-Goals/goals-for-2017-tw.png
    Created /Users/fpigeonjr/Code/myBlog/content/posts/2017-03-01-postCSS/postcss-fb.png
    Created /Users/fpigeonjr/Code/myBlog/content/posts/2017-03-01-postCSS/postcss-tw.png
    Created /Users/fpigeonjr/Code/myBlog/content/posts/2017-03-12-vuejs/vue-fb.png
    Created /Users/fpigeonjr/Code/myBlog/content/posts/2017-03-12-vuejs/vue-tw.png
    Created /Users/fpigeonjr/Code/myBlog/content/posts/2017-12-15-year-in-review/2017-year-in-review-fb.png
    Created /Users/fpigeonjr/Code/myBlog/content/posts/2017-12-15-year-in-review/2017-year-in-review-tw.png
    Created /Users/fpigeonjr/Code/myBlog/content/posts/2018-02-21-2018-goals-for-2018/goals-for-2018-fb.png
    Created /Users/fpigeonjr/Code/myBlog/content/posts/2018-02-21-2018-goals-for-2018/goals-for-2018-tw.png
    Created /Users/fpigeonjr/Code/myBlog/content/posts/2018-12-10-Gartner-takeaways/top3-gartner-takeaways-fb.png
    Created /Users/fpigeonjr/Code/myBlog/content/posts/2018-12-10-Gartner-takeaways/top3-gartner-takeaways-tw.png
    Created /Users/fpigeonjr/Code/myBlog/content/posts/2018-12-17-poormans-standing-desk/poormans-standing-desk-fb.png
    Created /Users/fpigeonjr/Code/myBlog/content/posts/2018-12-17-poormans-standing-desk/poormans-standing-desk-tw.png
    ✨  Done in 41.57s.

    ```

1. Next I ran `yarn deploy` to publish on [my blog](https://frankpigeon.com).

   I took a look at the twitter preview and it doesn't load the image.

   [https://twitter.com/fpigeonjr/status/1076101442048610306](https://twitter.com/fpigeonjr/status/1076101442048610306)

> Thanks for all the support and help. Happy Holidays!

**Comment by maxpou** [comment]
Hi Frank,
1. go to https://frankpigeon.com/goals-for-2018
2. check the source and see the generated url for this SEO tags:
You have few weird URL: 
* `﻿﻿<meta name="twitter:image" content="https://frankpigeon.co/static/books-882607c552b006b85ade1c788a16b2b3.jpg" data-react-helmet="true">`
* `<meta property="og:url" content="https://frankpigeon.co/goals-for-2018" data-react-helmet="true">`
* `<meta property="og:image" content="https://frankpigeon.co/static/books-882607c552b006b85ade1c788a16b2b3.jpg" data-react-helmet="true">`

URL is `frankpigeon.co` instead of `frankpigeon.com` (last **m** is missing).


Also, check how you populate the fields `imageTw` and `imageFb`. I took your post "goals-for-2018" as example (https://github.com/fpigeonjr/myBlog/blame/master/content/posts/2018-02-21-2018-goals-for-2018/index.md#L6-L7).
You set the cover file instead of the generated file.
It should be something like : 
* `imageTw: ./goals-for-2018-tw.png`
* `imageFb: ./goals-for-2018-fb.png`

I hope it helps you :)
Cheers
🎅

**Comment by fpigeonjr** [comment]
Thanks @maxpou this was very helpful in troubleshooting my site.

2. I resolved the SEO issues by updated in the SEO component by changing the following code

![Imgur](https://i.imgur.com/JVP7oVW.png)

I also updated all the `index.md` files for my posts to match the generated social media cards and now everything is working!

![Imgur](https://i.imgur.com/yazKiro.png)

Thanks so much for your assistance and wish you a Merry Christmas!
